src/test/java/com/artipie/docker/DockerProxyIT.java

```diff
@@ -41,75 +37,70 @@
  *  It seems that body is being read by some other entity in Artipie,
  *  so it requires investigation.
  *  Similar `CachingProxyITCase` tests works well in docker-adapter module.
+ *  @todo #996:30min Refactor set up of a test's steps.
+ *   Consider creating a class to set up steps of test using less verbose way.
```

Comment:
I'm not sure what exactly requires refactoring in set up - there is just one line.

Reply:
I added a code example of how it can look. 
So if you think this is excess, I'll remove the puzzle.